beta/style/beta.css

```diff
@@ -0,0 +1,16 @@
+/* Set Background slightly darker than white */
+#body {
+background-color: gray
+}
+
+/* Set Background color to transperent, for inheiritance reasons, trying to imitate windows app footer */
+a {
+background-color: transperent;
```

Comment:
You have a typo on this line, also please move the styling directly in index.html, there is no need for a new file and folder

Reply:
Gotcha, done